actionpack/lib/action_dispatch/system_test_case.rb

```diff
@@ -134,7 +134,7 @@ def self.start_application # :nodoc:
     #   driven_by :selenium, using: :firefox
     #
     #   driven_by :selenium, using: :headless_firefox
-    def self.driven_by(driver, using: :chrome, screen_size: [1400, 1400], options: {})
+    def self.driven_by(driver, using: nil, screen_size: [1400, 1400], options: {})
```

Comment:
Wy are we not using `chrome` by default here?

Reply:
@rafaelfranca i would say this i can revert back i just use **The Boy Scout Rule** it's not make sense since different drivers would register default browsers by default. Since Rails has dependency on Capybara capybara already registering drivers by it self check here:

https://github.com/teamcapybara/capybara/blob/master/lib/capybara.rb#L465-L475

@rafaelfranca also when i just remove default using e.g. `:chrome` option nothing happened e.g. all tests passed so i would say would be good to add test case when chrome is required (it's passing since capybara already register default driver). It's because browser specification is not required by Selenium driver registered by default in capybara.

One more thing: 
I would say default browser should be added here:

https://github.com/rails/rails/blob/master/actionpack/lib/action_dispatch/system_testing/browser.rb#L8-L21

Either as default option value (name = :chrome) or as default state for browser type. If you think is better.